### DISCUSSION TITLE
Extracting the dimensions of a rectangular area of interest without resizing.

### DISCUSSION THREAD

**Discussion by lm-sudo** [ImageMagick/ImageMagick#6880] [Help]
### ImageMagick version

6.9.10-23

### Operating system, version and so on

Ubuntu 20.04.6 LTS

### Description

Hello,

my goal is to extract the dimensions of a rectangle(label) that rests upon a uniformly colored background. This should be done in such a way that the dimensions of the image are not changed(I have tried cropping but naturally that resized the image). I have attached an image for example. In the example image I have attached, the label is completely blank, but there may be some color/text on the label. Retaining any of the color or text within the label is of no concern. 

Currently the only way I know to do this(and it is not full proof) would be to programmatically scan the page from top to bottom until I identify the corners of the label and then determine the height and width from that. I'm wondering if this is more easily accomplished with imagemagick? 

Thank you.
![88742781_20231121123032](https://github.com/ImageMagick/ImageMagick/assets/89110072/8e75c273-47e0-428f-8843-b36bc6ba1b7a)

**Comment by fmw42** [comment]
Your background is not uniformly colored!

What do you want for the output?  Can you show an example?

Perhaps what you want (on an image with uniform background) is:

`convert your_image.png -format "%@" info:`

Or

`convert your_image.png -trim new_image.png`

If you background is not perfectly uniform, add -fuzz XX% to allow a percent tolerance in color.
